### PR TITLE
Prepare library for digital twin intergration

### PR DESCRIPTION
**Pull request by LukeParky**
Remove all instances of harp.gl, replace with cesium.
Add base documentation
Add github workflow to verify code builds
Add github workglow to publish to NPM package manager

**Development Checklist**
 - [x] Make code change
 - [x] Update tests - Simple linting and verifying build done so far, more testing will come later
   - [x] Update / create new tests
   - [x] Ensure these tests have the expected behaviour
   - [x] Test locally and ensure tests are passing
 - [x] Ensure tests passing on GH Actions
 - [x] Update documentation
   - [x] Doc strings
   - [x] Readme
 - [x] Update package version

**Reviewer Checklist**
- [ ] Read documentation, check if it is readable and complete
- [ ] Check testing and whether it needs more scope
- [ ] Check code for code smells or structure problems

### CODE REVIEW

**Comment by LukeParky** [comment]
Hi @caseyli92 and @rosepearson. If either of you have time could you have a quick look at this PR, then I can finish up a small one for intergrating the library into the digital twin.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+DESCRIPTION OF PR:
+
```
Comment (rosepearson) [inline .github/pull_request_template.md:2]: Nice addition!

File: .github/workflows/compliance.yml [hunk 2]
```diff
@@ -1,33 +1,48 @@
...
-
-      - name: Install dependencies
-        run: npm ci
-      - name: Run build
+name: Pull Request Compliance
+on:
+  pull_request:
+
+jobs:
+  check-version:
```
Comment (rosepearson) [inline .github/workflows/compliance.yml:6]: Nice addition

File: .github/workflows/tag-and-release.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: Publish to npm
```
Comment (rosepearson) [inline .github/workflows/tag-and-release.yml]: Nice addition. Makes sense as a separate workflow to me - which is what you've done.

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (rosepearson) [inline LICENSE:1]: Good addition!

File: README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+# Geo Visualisation Components
+
```
Comment (rosepearson) [inline README.md:2]: Glad to see this added

File: dev/serve.vue [hunk 6]
```diff
@@ -20,7 +20,7 @@ export default Vue.extend({
   },
   data() {
     return {
-      hereApiToken: process.env.VUE_APP_HEREAPI,
+      cesiumAccessToken: process.env.VUE_APP_CESIUM_ACCESS_TOKEN,
     }
```
Comment (rosepearson) [inline dev/serve.vue:24]: I can't say I fully understand this but I think there is enough there I could google to understand. Good clear code is what I'm trying to say!

File: package.json [hunk 7]
```diff
@@ -55,6 +65,8 @@
     "minimist": "^1.2.5",
     "rimraf": "^3.0.2",
     "rollup": "^2.52.8",
+    "rollup-plugin-copy": "^3.4.0",
+    "rollup-plugin-import-css": "^3.0.3",
     "rollup-plugin-terser": "^7.0.2",
     "rollup-plugin-typescript2": "^0.30.0",
```
Comment (rosepearson) [inline package.json:71]: Looks good

**Review by rosepearson — APPROVED** [review]
Hi Luke, I've had a quick review for documentation and code smells. Looks good to me.

File: .github/workflows/tag-and-release.yml (hunk 3 above)
Comment (LukeParky) [reply .github/workflows/tag-and-release.yml]: Thanks, I just noticed while reading this that it only releases and not tags. this is intentional but I will rename the name of the file

File: dev/serve.vue (hunk 6 above)
Comment (LukeParky) [reply dev/serve.vue:24]: Thanks, that's good to hear. I find Vue is quite easy to read but its good to hear from someone that hasn't used it before

File: build/check-version.py [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+               self.patch == other.patch
+
+    def __lt__(self, other: "SemVer"):
+        return self != other and \
+               self.major <= other.major and \
+               self.minor <= other.minor and \
+               self.patch <= other.patch
+
+    def __repr__(self):
+        return f"SemVer('{self.major}.{self.minor}.{self.patch}"
```
Comment (caseyli92) [inline build/check-version.py:30]: add closing bracket: return f"SemVer('{self.major}.{self.minor}.{self.patch})"

Comment (caseyli92) [inline build/check-version.py]:
```suggestion
        return f"SemVer('{self.major}.{self.minor}.{self.patch})"
```

File: build/check-version.py [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+    def __eq__(self, other: "SemVer"):
+        return self.major == other.major and \
+               self.minor == other.minor and \
+               self.patch == other.patch
+
+    def __lt__(self, other: "SemVer"):
+        return self != other and \
+               self.major <= other.major and \
+               self.minor <= other.minor and \
+               self.patch <= other.patch
```
Comment (caseyli92) [inline build/check-version.py]:
```suggestion
               self.patch < other.patch
```

**Review by caseyli92 — CHANGES_REQUESTED** [review]
Just a couple of changes need to be made.

File: build/check-version.py (hunk 8 above)
Comment (LukeParky) [reply build/check-version.py]:
```suggestion
        return f"SemVer('{self.major}.{self.minor}.{self.patch}')"
```
Good spot, but I think this is what we both mean

**Review by caseyli92 — APPROVED** [review]

**Pull request merged**